### PR TITLE
Support for Swift Package Manager

### LINKED ISSUE #112: [iOS] Swift Package Manager support
Already implemented in #106. Since SPM doesn't support UIKit imports, I had to `#if os(iOS)` bunch of code, which is not ideal. We have to get back to this once SPM supports iOS builds.

### PR DESCRIPTION
**Pull request by kovpas**
Had to `#if os(iOS)` bunch of UIKit stuff, since SPM doesn't support it yet.

Resolves #112

### CODE REVIEW

File: ios/RIBs/Classes/LaunchRouter.swift [hunk 1]
```diff
@@ -47,3 +49,5 @@ open class LaunchRouter<InteractorType, ViewControllerType>: ViewableRouter<Inte
         load()
     }
 }
+
+#endif
```
Comment (artman) [inline ios/RIBs/Classes/LaunchRouter.swift]: Newline please

Comment (kovpas) [reply ios/RIBs/Classes/LaunchRouter.swift]: Done

**Comment by kovpas** [comment]
> I don't like the idea that we have to ifdef our code to make it work with Swift package manager. Maybe we should wait for Swift package manager to support platforms before we add support for it?

Yeah, I'm leaving the decision up to you. In other opensource projects, people are usually asking about SPM support. Let's monitor if that's gonna be the case with RIBs and get back to the topic if it is.

**Comment by alanzeino** [comment]
I don't think this is needed. SPM can't even build iOS projects yet.

**Comment by kovpas** [comment]
Yes, good point. Closing the PR.

**Pull request closed without merging**